_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -1237,6 +1243,41 @@ mfxStatus VAAPIVideoProcessing::Execute(mfxExecuteParams *pParams)
         break;
     }
 
+#if (MFX_VERSION >= 1025)
+        uint8_t& chromaSitingMode = m_pipelineParam[0].input_color_properties.chroma_sample_location;
+        chromaSitingMode = VA_CHROMA_SITING_UNKNOWN;
```

Comment:
should be in switch, I suppose

Reply:
Also you should add a reference to https://github.com/Intel-Media-SDK/MediaSDK/issues/1737 via:
Fixes: # 1737

Reply:
It can be left here - this is just local reference, it will be destroyed after exiting function scope.

Reply:
"Fixes" issue is addressed